actions/fragment.py

```diff
@@ -6,7 +6,7 @@
 
 
 class FragmentAction(Action):
-    def __init__(self, environment_id=None, correct_order=None, fragsize=-1, segment=True):
+    def __init__(self, environment_id=None, correct_order=None, fragsize=-1, segment=True, overlap=0):
```

Comment:
can you merge in master?

Reply:
wait a minute is overlapping segmentation in master rn?

Reply:
ping @Kkevsterrr

Reply:
ping @Kkevsterrr

Reply:
Not in this master yet, I’ll message you